File: my-app/src/app/common/standing-dto.ts
import {ClubDto} from "./club-dto";
import {SeasonDto} from "./season-dto";

export interface StandingDTO {
  id:number,
  club:ClubDto,
  matchesPlayed:number,
  wonPlayed:number,
   drawMatches:number,
   lostMatches:number,
   goalScored:number,
   goalConceded:number,
   points:number,
   seasonId:SeasonDto
}

File: FootballApp/src/main/java/com/football/dev/footballapp/dto/StandingDTO.java
package com.football.dev.footballapp.dto;

public record StandingDTO(
  Long id,
  ClubDto club,
  Integer matchesPlayed,
  Integer wonMatches,
  Integer drawMatches,
  Integer lostMatches,
  Integer goalScored,
  Integer goalConceded,
  Integer points,
  SeasonDto seasonId
) {}
